package.json

```diff
@@ -80,6 +80,7 @@
     "apollo-server-errors": "^3.3.0",
     "apollo-server-types": "^3.8.0",
     "cloudinary": "^1.41.3",
+    "cockatiel": "^3.2.1",
```

Comment:
Decided to use this lib at the end since https://github.com/nodeshift/opossum has quite worse types and options wise is quite limited. For example it would not work with instance based clients as we have now since it requires a function to be passed at the moment of breaker initialization.